cmd/preferredimports/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
We will need to change this file. See: OWNERS

Reply:
owner changed